tests/test_warning.py

```diff
@@ -86,13 +83,12 @@ def test_currentphenomenons_with_coastal_bulletin(dep: str, res: bool) -> None:
         domain=dep, depth=1, with_costal_bulletin=True
     )
     has_coastal_phenomenom = any(
-        phenomenom["phenomenon_id"] == 9
+        phenomenom["phenomenon_id"] == "9"
```

Comment:
mypy issue here, maybe still an int ?

Reply:
I updated my code. The phenomenon id returned by meteo france is now a character string. There should be no more typing errors.